bin/package-electron

```diff
@@ -50,10 +50,10 @@ yarn workspace desktop-electron update-client
         if [ -f ../../.secret-tokens ]; then
             source ../../.secret-tokens
         fi
-        yarn build --publish never --arm64 --x64
+        yarn build
```

Comment:
Removing the overrides to the build params, the build script sets it all correctly